### PR TITLE
Fixed README to link the correct Clonkk github profile

### PR DESCRIPTION
**Pull request by Clonkk**
The README was pointing some poor soul who happens to use the same name :smile:

### CODE REVIEW

**Comment by pietroppeter** [comment]
oops... thanks for fixing this! 😄

actually in order to merge ideally you should change the index.nim file and run nim r docs\index (so that it will be correctly regenerate both the index.html and README.md file, you will need also to install nimoji for that). If you want to try, you are most welcome, otherwise let me know and I will do it!

**Comment by Clonkk** [comment]
Ah I didn't see you were generating the README file.

**Comment by pietroppeter** [comment]
thanks, what you did is correct. the CI is failing because the testing framework has some bugs:

* as a first issue the CI it is not clear what the issue is (I should improve on that).
* anyway I think the issue is the fact that when you generate the document the repo link in top right of the document goes to your fork and not to the upstream repo.

for the moment a manual fix of the problematic line could be done (I will add a note in the code), but I have to think of something better to improve UX for contributors...

File: docs/index.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
...
+  <span><a href="https://github.com/Clonkk/nimib"><svg aria-hidden="true" width="1.2em" height="1.2em" style="vertical-align: middle;" preserveAspectRatio="xMidYMid meet" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59c.4.07.55-.17.55-.38c0-.19-.01-.82-.01-1.49c-2.01.37-2.53-.49-2.69-.94c-.09-.23-.48-.94-.82-1.13c-.28-.15-.68-.52-.01-.53c.63-.01 1.08.58 1.23.82c.72 1.21 1.87.87 2.33.66c.07-.52.28-.87.51-1.07c-1.78-.2-3.64-.89-3.64-3.95c0-.87.31-1.59.82-2.15c-.08-.2-.36-1.02.08-2.12c0 0 .67-.21 2.2.82c.64-.18 1.32-.27 2-.27c.68 0 1.36.09 2 .27c1.53-1.04 2.2-.82 2.2-.82c.44 1.1.16 1.92.08 2.12c.51.56.82 1.27.82 2.15c0 3.07-1.87 3.75-3.65 3.95c.29.25.54.73.54 1.48c0 1.07-.01 1.93-.01 2.2c0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z" fill="#000"></path></svg></a></span>
```
Comment (pietroppeter) [inline docs/index.html:38]: this is the part that should be reverted manually and the CI should pass

Comment (Clonkk) [reply docs/index.html]: Weird one, but I'm not surprised by anything anymore when CI are involved :smile:

Comment (pietroppeter) [reply docs/index.html]: well, it is a bit weird the fact that I am committing the full html (in principle I could have a docs building CI), but for the moment I want to have complete control over the default output. It is also not so common the fact that I am inserting by default the link to the repo in the document, but I kind of like to have this as default behaviour. it is really not CI fault, but mine ;)

File: docs/index.html [hunk 2]
```diff
@@ -27,7 +27,7 @@
   display:none
 }
 </style>
-  
+
```
Comment (pietroppeter) [inline docs/index.html:30]: this should be one whitespace issue which might create problems with the CI

File: docs/index.html [hunk 3]
```diff
@@ -568,4 +568,4 @@ <h3>why the Q &amp; A?</h3>
   }
 }
 </script></body>
-</html>
\ No newline at end of file
+</html>
```
Comment (pietroppeter) [inline docs/index.html:571]: and this should be a second whitespace issue which might create problems with the CI

**Comment by pietroppeter** [comment]
damn, now CI is being very picky (two whitespace issues and I have no idea where they come from). I really feel bad about it.

**Comment by Clonkk** [comment]
Ah, I use neovim and it automatically trim whitespace in file when I save them. Could be that ?

**Comment by pietroppeter** [comment]
> Ah, I use neovim and it automatically trim whitespace in file when I save them. Could be that ?

yes, for the end of file whitespace it is most likely that. Maybe it also removed the other whitespace which appears mid document (which I have no clue why is there...)?

**Comment by Clonkk** [comment]
Closing for a cleaner branch https://github.com/pietroppeter/nimib/pull/42

**Pull request closed without merging**